sympy/solvers/solvers.py

```diff
@@ -1539,7 +1539,7 @@ def _solve(f, *symbols, **flags):
         msg = ''  # there is no failure message
 
         # Poly is generally robust enough to convert anything to
-        # a polynomial and tell us the different generators that it
+        # a p   olynomial and tell us the different generators that it
```

Comment:
Typo?

Reply:
Yes I plan to fix this with my next update.